src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -328,6 +328,26 @@
             -n MyHttpSettings --probe MyNewProbe
 """
 
+helps['network application-gateway identity'] = """
+type: group
+short-summary: Manage the identity information
```

Comment:
Manage service identity for network application-gateway?

Reply:
yes. I fixed this one but have not pushed. Thanks.